app/src/main/java/com/provider/exception/handler/GlobalExceptionHandler.java

```diff
@@ -45,6 +47,22 @@ protected ResponseEntity<ErrorResponse> handleJakartaConstraintViolationExceptio
     return new ResponseEntity<ErrorResponse>(errorResponse, HttpStatus.BAD_REQUEST);
   }
 
+  @ExceptionHandler
+  protected ResponseEntity<ErrorResponse> handleNoContentException(NoContentException ex) {
+    String errorMessage =
+        "value not expected, orderBy can be \"created_at\" and order can be \"asc\" or \"desc\"";
+    ErrorResponse errorResponse = new ErrorResponse().ok(false).errorMessage(errorMessage);
+    return new ResponseEntity<ErrorResponse>(errorResponse, HttpStatus.NO_CONTENT);
+  }
+
+  @ExceptionHandler
+  protected ResponseEntity<ErrorResponse> test(MethodArgumentTypeMismatchException ex) {
+    String errorMessage =
+        "value not expected, orderBy can be \"created_at\" and order can be \"ASC\" or \"DESC\"";
+    ErrorResponse errorResponse = new ErrorResponse().ok(false).errorMessage(errorMessage);
```

Comment:
Try to take the message from the exception, I suppose it should be detailed enough